### ISSUE TITLE
Error log if there are no applications

### ISSUE THREAD

**Issue by f1sh1918** [digitalfabrik/entitlementcard#1223]
**Describe the bug**
Receive an error log when fetching data for `/application` and `/verification`

**To Reproduce**
1. create an application
2. click on the mail link to view your application
3. remove some chars from the magic link
4. Error can be found in the backend log

**Expected behavior**

**LOG**
```js
Nov 22 13:46:39 entitlementcard.tuerantuer.org backend[120716]: [JettyServerThreadPool-620] ERROR app.ehrenamtskarte.backend.common.webservice.GraphQLExceptionHandler - Exception while fetching data (/application) : null
```

**Comment by michael-markl** [comment]
> @sarahsporck the only place where we use /application is for the applicant view attachment

I am pretty sure, this log does not come from the application attachment endpoint (because this would not generate a graphql error afaics).
The `/application` in the graphql error message says, that the graphql endpoint could not retrieve the property `application` of the specified query/mutation.
Queries that have this property:
* https://github.com/digitalfabrik/entitlementcard/blob/main/administration/src/graphql/applications/getApplication.graphql#L2
* https://github.com/digitalfabrik/entitlementcard/blob/main/administration/src/graphql/applications/getApplicationByApplicant.graphql#L2

The solution would be to find where the exception is thrown and instead do a proper error handling, I think?

**Comment by f1sh1918** [comment]
it is thrown when the passed `accessKey` can't be found in the database and that happens if the a broken link was opened

**Comment by f1sh1918** [comment]
i just updated the steps to reproduce.
This error is probably connected with broken links the user may receive via mail.
I gonna block this issue due to this pr https://github.com/digitalfabrik/entitlementcard/pull/1378 which maybe solves the issue or at least let it less occur.

**Comment by f1sh1918** [comment]
we still have link 60-80 logs within 24hours. Probably due to broken link. We could check if the `accessKey` length is valid

**Comment by f1sh1918** [comment]
are there any opinions to that @michael-markl , @ztefanie or @seluianova

**Comment by michael-markl** [comment]
Is this the only thing that is logged? 
```
Nov 22 13:46:39 entitlementcard.tuerantuer.org backend[120716]: [JettyServerThreadPool-620] ERROR app.ehrenamtskarte.backend.common.webservice.GraphQLExceptionHandler - Exception while fetching data (/application) : null
```

**Comment by f1sh1918** [comment]
yes we could maybe try to add a log for the `accessKey` that is used

**Comment by f1sh1918** [comment]
added follow up issues
https://github.com/digitalfabrik/entitlementcard/issues/1862

https://github.com/digitalfabrik/entitlementcard/issues/1871

so this can be closed
